### ISSUE TITLE
The SpongeBob SquarePants Movie [54510027]

### ISSUE THREAD

**Issue by Fisherman166** [Cxbx-Reloaded/game-compatibility#620]
### Description
[Wikipedia](https://en.wikipedia.org/wiki/The_SpongeBob_SquarePants_Movie_(video_game))

Game crashes immediately with no audio or video.

Received Exception Code 0xC0000005 @ EIP := 0x001756B6(=unknown+0x0)

### Dumps 
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1689673/Xbe.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1689675/KrnlDebug.txt)
[The SpongeBob SquarePants Movie-6fc8ce98.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1689676/The.SpongeBob.SquarePants.Movie-6fc8ce98.txt)

### Status
**Previous behavior:** 
None Reported.

**Current behavior:**
Crashes immediately with no audio or video.

### Screenshots
None

### Additional Information
Ran on commit c08c4d922458b737b191a936355cd59366b6f567 (Feb 02, 2018)

**Comment by jthecodebuilder** [comment]
Seems to run now, but in game graphics are broken (no character models, and camera viewport rendering is broken), and there appears to be a **memory leak** or something that causes a out of memory error (CreateImageSurface failed with E_OUTOFMEMORY). But I have plenty of memory (8 GB)...

**Comment by jthecodebuilder** [comment]
Now rendering past the initial loading screen seems to be completely broken... (black screen, even at movie clips)

**Comment by jthecodebuilder** [comment]
Now it only plays the first logo video (with only audio), and doesn't seem to play the second one. Also, here is a KrnlDebug log file.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2041128/KrnlDebug.txt)

**Comment by jthecodebuilder** [comment]
Now it seems to do crash again, with a message of: 
`Received Exception Code 0xC0000005 @ EIP := 0x04067772`
However, it seems to still be able to run on the LLE GPU emulation, but i forgot to mention it now renders much worse now using it (horizontially i seem to be seeing)..

I also forgot to mention that it getting stuck on the first video is now fixed, however I'm pretty sure the rendering has not been fixed past that point. 
Here is a [KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2070390/KrnlDebug.txt) to make things clearer. 

So, basically, it might be something with the HLE GPU stuff.
